libs/remix-lib/src/execution/txListener.js

```diff
@@ -9,6 +9,16 @@ const defaultExecutionContext = require('./execution-context')
 const txFormat = require('./txFormat')
 const txHelper = require('./txHelper')
 
+function addExecutionCosts(txResult, tx) {
```

Comment:
This function is already defined in this file, not sure which one you want to keep